atlasdb-config/src/main/java/com/palantir/atlasdb/http/NotCurrentLeaderExceptionMapper.java

```diff
@@ -29,6 +28,6 @@
 public class NotCurrentLeaderExceptionMapper implements ExceptionMapper<NotCurrentLeaderException> {
     @Override
     public Response toResponse(NotCurrentLeaderException exception) {
-        return Response.noContent().status(503).header(HttpHeaders.RETRY_AFTER, "0").build();
+        return ExceptionMappers.encode503ResponseWithRetryAfter(exception);
```

Comment:
Isnt this still `noContent` - not sure if entity is content but I checked and is not clear from the API?